new_devsurvey/src/components/TableComponent.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
This component is unfinished

Reply:
Please explain better

Reply:
The best thing would be to finish a component before pushing, but here, it seems that the diagram has not been completed